app/components/viral/pagy/limit_component.html.erb

```diff
@@ -1,11 +1,9 @@
-<div
-  class="inline-flex items-center mt-4 space-x-2  text-slate-500 dark:text-slate-400"
->
+<%= render Viral::BaseComponent.new(**system_arguments) do %>
```

Comment:
**nitpick:** Sorry can you just revert the changes to these two files and just add the id directly to the div.  I think it will be easier to read.  The div does not need any extra things passed into it.

Reply:
6a47e048399f7821455682eb0156e895003c9eaf